docs/docs/deploy-with-docker.md

```diff
@@ -1,14 +1,71 @@
 # Deploy with Docker
 
-You can use Docker to deploy Streamsync anywhere. If you're an experienced Docker user, you may want to go straight to the provided Dockerfile.
+You can use Docker to deploy Writer anywhere. If you're an experienced Docker user, you may want to go straight to the provided Dockerfile.
 
 ## Creating a Docker image
 
 ### Setting up
 
 - Make sure you have Docker installed.
 - Open a terminal and navigate to your app's folder.
-- Create a `pyproject.toml` using `poetry init` and install `streamsync` using `poetry add streamsync`
+- Create a `pyproject.toml` using `poetry init` and install `writer` using `poetry add writer`
+
+### What is Poetry and Its Purpose?
```

Comment:
@raaymax @anant-writer I'd phrase more like "Poetry is [...], and Writer Framework uses it to [...]." It needs to be clear that it isn't an option.